lib/page-object/page_populator.rb

```diff
@@ -58,9 +58,9 @@ def populate_radiobuttongroup(key, value)
       return self.send("select_#{key}", value)
     end
 
-  def populate_select_list(key, value)
+    def populate_select_list(key, value)
```

Comment:
Are you asking why this page isn't implemented with code like this: `self.send("#{key}=", value)`? The code that was there until a couple months ago was: `self.send("#{key}_element").select(value)`. Regardless I think the rightest answer is to change how Watir behaves in `Select#select`; working on that code now. So all of this may be unnecessary in the next minor Watir release. :)